README.md

```diff
@@ -1,109 +1,211 @@
- - `terraform apply`
+All responses are JSON unless otherwise noted. The POST endpoint requires a valid JWT.
+
+| Method | Endpoint | Auth | Description |
+|--------|----------|------|-------------|
+| GET | `/api` | none | Returns OpenAPI 3 definition (raw YAML) |
+| GET | `/token` | none | 303 redirect to external token provider |
+| GET | `/topics` | none | Lists available topic names |
+| GET | `/topics/{topicName}` | none | Returns JSON Schema for the topic |
+| POST | `/topics/{topicName}` | JWT | Validates + forwards message to configured sinks |
+| POST | `/terminate` | (internal) | Forces Lambda process exit (used to trigger cold start & config reload) |
+
+Status codes:
+- 202 – Accepted (all writers succeeded)
+- 400 – Schema validation failure
+- 401 – Token missing/invalid
+- 403 – Subject unauthorized for topic
+- 404 – Unknown topic or route
+- 500 – One or more writers failed / internal error
+
+## Configuration
+All core runtime configuration is driven by JSON files located in `conf/` unless S3 paths are specified.
+
+Primary file: `conf/config.json`
+Example (sanitized):
+```json
+{
+  "access_config": "s3://<bucket>/access.json",
+  "token_provider_url": "https://<token-ui.example>",
+  "token_public_key_url": "https://<token-api.example>/public-key",
+  "kafka_bootstrap_server": "broker1:9092,broker2:9092",
+  "event_bus_arn": "arn:aws:events:region:acct:event-bus/your-bus"
+}
+```
+Supporting configs:
+- `access.json` – map: topicName -> array of authorized subjects (JWT `sub`). May reside locally or at S3 path referenced by `access_config`.
+- `topic_*.json` – each file contains a JSON Schema for a topic. In the current code these are explicitly loaded inside `event_gate_lambda.py`. (Future enhancement: auto-discover or index file.)
+- `api.yaml` – OpenAPI spec served verbatim at runtime.
+
+Environment variables:
+- `LOG_LEVEL` (optional) – defaults to `INFO`.
+
+## Deployment
+Infrastructure-as-Code examples live in the `terraform/` directory. Variables are supplied via a `*.tfvars` file or CLI.
+
+### Zip Lambda Package
+Use when no custom native libraries are needed.
+1. Run packaging script: `scripts/prepare.deplyoment.sh` (downloads deps + zips sources & config)
+2. Upload resulting zip to S3
+3. Provide Terraform variables:
+   - `aws_region`
+   - `vpc_id`
+   - `vpc_endpoint`
+   - `resource_prefix` (prepended to created resource names)
+   - `lambda_role_arn`
+   - `lambda_vpc_subnet_ids`
+   - `lambda_package_type = "Zip"`
+   - `lambda_src_s3_bucket`
+   - `lambda_src_s3_key`
+4. `terraform apply`
+
+### Container Image Lambda
+Use when Kafka access needs Kerberos / SASL_SSL or custom `librdkafka` build.
+1. Build image (see comments at top of `Dockerfile`)
+2. Push to ECR
+3. Terraform variables:
+   - Same networking / role vars as above
+   - `lambda_package_type = "Image"`
+   - `lambda_src_ecr_image` (ECR image reference)
+4. `terraform apply`
+
+## Local Development & Testing

```

Comment:
Here I would only mention chapters in DEVELOPER.md and link them. You are here make doc duplicity.